src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -11,15 +12,28 @@ public class ReadWriteList<E> {
 
     public void add(E element) {
```

Comment:
Only one thread can update the data at a time causing other threads (both readers and writers) to block until the update is over.